c7n/policy.py

```diff
@@ -478,6 +488,14 @@ def provision(self):
         with self.policy.ctx:
             self.policy.log.info(
                 "Provisioning policy lambda %s", self.policy.name)
+
+            policy = {
```

Comment:
naming this as policy is perhaps a little confusing, config/vars/etc would be more apt.